### ISSUE TITLE
javascript inline issue

### ISSUE THREAD

**Issue by Russell-Allen** [thymeleaf/thymeleaf#200]
(using version 2.0.17)
The following fails to produce valid javascript:

```
<script th:inline="javascript">
  var data = {
    id: /*[[${foo}]]*/ '12345678-1234-1234-1234-1234567890ab'
  };
</script>
```

The value of foo is injected and the prototype value is left on the line.  If I move the closing of the folloing line onto the end, then the closing of the JSON object is removed.

Note this is a simplified example.  This is actually nested inside a function, which only adds more trailing characters, such as closing parenthesis, etc.

**Comment by danielfernandez** [comment]
I can confirm this is already solved in 2.0.18-SNAPSHOT, and therefore will be solved in the 2.0.18 release.

**Comment by Russell-Allen** [comment]
Thank you!  I'll re-post validation once 2.0.18 hits GA.
